qiskit_nature/results/protein_folding_result.py

```diff
@@ -60,3 +91,27 @@ def get_result_binary_vector(self) -> str:
             result.append(self._best_sequence[index])
 
         return "".join(result[::-1])
+
+    def save_xyz_file(self, name: str = None, path: str = "") -> None:
+        """
+        Generates a .xyz file.
+        Args:
+            name: Name of the file to be generated.
+            path: Path where the file will be generated.
+        """
+        if name is None:
+            name = str(self._peptide.get_main_chain.main_chain_residue_sequence)
+        self.protein_shape_file_gen.save_xyz_file(name=name, path=path)
+
+    def plot_folded_protein(
+        self, title: str = "Protein Structure", ticks: bool = True, grid: bool = False
+    ) -> None:
+        """
+        Plots the molecule in 3D.
+        Args:
+            title: The title of the plot.
+            ticks: Boolean for showing ticks in the graphic.
+            grid: Boolean for showing the grid in the graphic.
```

Comment:
I guess it would make more sense to allow plotting customization with an arbitrary `kwargs` dictionary, rather than exposing a limited subset of things, here.